doc/whats_new.rst

```diff
@@ -254,7 +254,7 @@ New features
    - The new module :mod:`nilearn.connectome` now has class
      :class:`nilearn.connectome.ConnectivityMeasure` can be useful for
      computing functional connectivity matrices.
-   - The function :func:`nilearn.connectome.sym_to_vec` in same module
+   - The function :func:`nilearn.connectome.sym_matrix_to_vec` in same module
```

Comment:
Yes, because changes you made in a section is released. New section should be added and included in what ever might be useful to users to refer.